config/sync/core.extension.yml

```diff
@@ -215,12 +215,13 @@ module:
   externalauth: 10
   menu_link_weight: 10
   stanford_profile_helper: 10
+  sdss_profile_helper: 10
```

Comment:
Not a requirement, but you could have this be higher weight than `stanford_profile_helper` so that any identical hooks come after.